mock-belastingdienst/index.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express')
+var app = express()
+
+app.get('/belastingdienst/bsn/995671239', (req, res) => {
+    res.writeHead(200, { 'Content-Type': 'application/json' });
+    res.write(JSON.stringify({
+        "_embedded": {
+            "gegevensbelastingdienst": [
+                {
+                    "burgerservicenummer": "995671239",
+                    "naam": {
+                        "voornamen": "Jan Hendrik",
+                        "voorletters": "J.H.",
+                        "geslachtsnaam": "Jong",
+                        "voorvoegsel": "de"
+                    },
+                    "geboortedatum": "17-06-1989",
+                    "alimentatieverplichtingen": "geen",
+                    "verplichtingenAndersDanBekendBijBKR": "geen",
+                    "inkomenUitOnderneming": "geen",
+                    "woningEnAndereOnroerendeZaken": "geen",
+                    "fiscaalPartner": "geen",
+                    "eigenBijdragePensioen": "56,73 euro"
```

Comment:
We might consider making this a little more API-like. Stuff like replacing "geen" with "false", and splitting the euro's into a compound structure with wholes/cents/currency. That makes the wp-service a little more complex, so I'm good with putting this down as a future issue if we have time.